src/main.js

```diff
@@ -31,13 +27,19 @@ function getRandomIndex(array) {
 }
 
 
+function getRandomCover() {
+  coverImage.src = covers[getRandomIndex(covers)];
+  coverTitle.innerText = titles[getRandomIndex(titles)];
+  tagline1.innerText = descriptors[getRandomIndex(descriptors)];
+  tagline2.innerText = descriptors[getRandomIndex(descriptors)];
+}
+getRandomCover()
```

Comment:
let's fix this indentation next iteration

Reply:
thanks, will do for sure